### ISSUE TITLE
Game problem

### ISSUE THREAD

**Issue by seb9797** [dimok789/loadiine_gx2#101]
Hello,

My wii : 5.5.1 

Loadiine gx works (test : yoshi, star fox guard, snoopy, etc..) but i can't save my progression...

Anyone have an idea why ? (ma sd card isn't lock...)

Thanks

Seb (France)

**Comment by CreeperMario** [comment]
Is there a `saves` folder on the root of your SD Card? If so, look inside and see if you can find some `8000000𝒙` and `common` folders, as these are where Loadiine puts save data. If you can't find any saves, report back, as this is actually a problem.

**Comment by seb9797** [comment]
I don't have a save folder on the root of my sd card... I must create a folder ?

Envoyé de mon iPhone

**Comment by CreeperMario** [comment]
Maybe. What happens when a game tries to save/load something? Does the game itself give an error?

**Comment by seb9797** [comment]
No, and i can see the save... But when i reboot, the save doesn't appear....

Bien cordialement,

S.ANQUETIL


Envoyé de mon iPhone

**Comment by seb9797** [comment]
anyone have an idea ?

**Comment by drizzt** [comment]
try to change your SD or to format it with 64k clusters

**Comment by juanfuentes95** [comment]
Loadiine create "save" folder automatically. Try change the format cluster (uncheck quick format!) or change the SD adapter.

**Comment by motoconcho2** [comment]
hey i got the same problem I´m going to change the adapter or uncheck quick format. if that work i will let you know if sometime of that work.

**Comment by motoconcho2** [comment]
is the adapter

**Comment by CreeperMario** [comment]
@motoconcho2 so your SD Card adapter was the problem for you, and changing your SD Card fixed it?
@seb9797 are you using the latest nightly build of Loadiine GX2? Perhaps there was a regression in an earlier build that nobody picked up on.

**Comment by motoconcho2** [comment]
@CreeperMario yes. I only change the adapter and that fixed my problem.

**Comment by astraldata** [comment]
actually, I had this issue -- apparently loadiine didn't create a save folder automatically (or it's automatic save-folder-making is broken on 5.5.1) but when I created one manually, it seems to work fine now.

**Comment by CreeperMario** [comment]
@astraldata when I first used Loadiine GX2, I used the Definitive Package from somebody on GBATemp, and it provided a saves folder created already (although it was blank).

**Comment by weso18** [comment]
I'm having the same problem with the games saving. Is Loadiine GX2 v0.2 the latest version? If not, what is the latest version that I should be using? The file structure that I'm using is: \wiiu\saves\nameofgame\u & \wiiu\saves\nameofgame\c  Is this correct? Loadiine did not automatically create folders so I created these myself. Should I be using \wiiu\saves\nameofgame\80000001 and \wiiu\saves\nameofgame\common instead? Others have mentioned that a different SD card, or changing the adapter has worked for them but before I go buy another one I want to make sure that I have everything else setup correctly, and that I am actually using the latest version of Loadiine. Thanks.

**Comment by hradec** [comment]
I'm actually seeing a similar issue... I have a bunch of save games that where working fine in loadiine, but after I took the SD card to my computer to add some roms to an emulator, I did an fsck (in linux), and it correct one error found. 

after this, when I launch any game from loadiine, it can see the saved data... all games start clean like there was no savegame. 

it's like the /wiiu/save doesn't exist or its empty, but the folder does exists and all the save games are in there... not sure what's going on!! 

any ideas?

**Comment by hradec** [comment]
Turns out that, based on the idea of creating a clean `sd:/wiiu/saves` folder, I made a backup copy of my current `sd:/wiiu/saves` folder from the SD Card (the one that loadiine was ignoring), moved the SD card `sd:/wiiu/saves` to `sd:/wiiu/saves.bak`, and copied back the backed up `saves/*`  into the SDCard sd:/wiiu/saves/.

essentially, I re-created the `sd:/wiiu/saves` folder, and copied all my saves into it! 

Now loadiine is "seeing" my saves correctly again and I can load then up in the games without problem!!! 

If someone runs into this problem, that's how to fix it! ;)

**Comment by Masamune3210** [comment]
Probably had some minor corruption in the filesystem relating to the folder. Recreating the folder is usually a good first step
